cpp/ql/lib/semmle/code/cpp/security/Encryption.qll

```diff
@@ -64,7 +64,7 @@ predicate isInsecureEncryption(string name) { name.regexpMatch(getInsecureAlgori
  */
 bindingset[name]
 predicate isEncryptionAdditionalEvidence(string name) {
-  name.toUpperCase().matches("%" + ["CRYPT", "CODE", "CODING", "CBC", "KEY", "CIPHER", "MAC"] + "%")
+  name.regexpMatch("(?i).*" + ["crypt", "code", "coding", "cbc", "key", "cipher", "mac"] + ".*")
```

Comment:
If we're going the regexp route, I believe it would be more efficient to have just one regexp:
```suggestion
  name.regexpMatch("(?i).*(crypt|code|coding|cbc|key|cipher|mac).*")
```
(this made quite a big difference in a more extreme case I investigated recently)

Reply:
Oh, yes. I completely forgot about `|` 🤦. I think your solution is probably a lot better.